src/views/core/layout/layout.view.tsx

```diff
@@ -1,14 +1,30 @@
 import { FC } from "react";
 
 import useLayoutStyles from "src/views/core/layout/layout.styles";
+import Snackbar from "src/views/shared/snackbar/snackbar.view";
+import { useGlobalContext } from "src/contexts/global.context";
+import { reportError } from "src/adapters/error";
 
 const Layout: FC = ({ children }) => {
   const classes = useLayoutStyles();
+  const { snackbar, closeSnackbar } = useGlobalContext();
+
+  const onCloseSnackbar = closeSnackbar;
+  const onReportFromSnackbar = reportError;
```

Comment:
I don't feel the need to do this, but I wanted to avoid the PR comment about the missing `on` prefix on the handlers.

We can go 3 ways regarding this:

 1. Reassigning the handlers as it's now.
 1. Naming the handlers with the `on` prefix at the source (GlobalContext)
 1. Accepting `onClose={closeSnackbar}` and `onReport={reportError}`

I'm ok with the three.